htdocs/core/boxes/box_birthdays.php

```diff
@@ -85,19 +85,19 @@ public function loadBox($max = 20)
 		if ($user->rights->user->user->lire) {
 			$tmparray = dol_getdate(dol_now(), true);
 
-			$sql = "SELECT u.rowid, u.firstname, u.lastname, u.birth as datea, 'birth' as typea, u.email, u.statut as status";
+			$sql = "SELECT u.rowid, u.firstname, u.lastname, u.birth as datea, date_format(u.birth, '%d') as daya, 'birth' as typea, u.email, u.statut as status";
```

Comment:
Why do you need the date_format() ? Using this make the database not able to use the index.
Why not just 
u.birth as daya ?